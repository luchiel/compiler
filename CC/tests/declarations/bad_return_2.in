void main()
{
return 5;
}