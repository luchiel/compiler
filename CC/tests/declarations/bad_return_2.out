Error: ParseError:3:9 Function type and value type in return statement do not match
